src/common/verbose.cpp

```diff
@@ -85,7 +85,12 @@ static constexpr char verbose_version[] = "v1";
 
 static setting_t<uint32_t> verbose {0};
 
-void print_header(const filter_status_t &filter_status) noexcept {
+// Component filters help manage verbose output by parsing and printing for
+// matching components. The filter status is tracked from verbose initializaton,
+// allowing queries for the component type during verbose printing.
+static filter_status_t filter_status;
```

Comment:
Having an object wrapped in a function in a global space is a better practice as it won't require initialization if not requested.

```suggestion
filter_status_t &filter_status() {
    static filter_status_t filter_status;
    return filter_status;
}
```

Reply:
Thanks! Fixed in the latest commits.